assets/app/views/projects.html

```diff
@@ -1,11 +1,11 @@
-<div>
+<div class="container">
   <div>
     <h1 style="margin-top: 10px;">Projects</h1>
     <div ng-if="$.isEmpty(projects)">
       You have no projects. For an example run <code>openshift kubectl create -f examples/hello-openshift/hello-project.json</code>
     </div>
     <div class="tile tile-click" ng-click="tileClickHandler($event)" style="margin-bottom: 10px;" ng-repeat="project in projects">
-      <h3><a class="tile-target" href="/project/{{project.metadata.name}}">{{project.displayName || project.metadata.name}}</a></h3>
```

Comment:
I thought urls were supposed to start with a `/`... am I remembering that wrong?